game/theater/theatergroundobject.py

```diff
@@ -621,8 +633,6 @@ def symbol_set_and_entity(self) -> tuple[SymbolSet, Entity]:
 
 
 class IadsBuildingGroundObject(BuildingGroundObject):
-    def mission_types(self, for_player: bool) -> Iterator[FlightType]:
-        from game.ato import FlightType
-
-        if not self.is_friendly(for_player):
-            yield from [FlightType.STRIKE, FlightType.DEAD]
```

Comment:
This restricts flights types to Strike and DEAD only! Just consider it a building like any other because while it's a part of the entire air defense system, a factory or a comms tower on their own are not an air defense...

Reply:
I'm guessing the intent was to exclude BAI rather than to exclude all the escort tasks and similar. This makes BAI valid again which is wrong. I'm not sure what the right fix is off the top of my head, but we shouldn't merge this. This is what I mean when I keep saying "We can't merge half a PR". All the front-end changes appear fine but other parts need to be changed (I'm not sure if the front-end changes are coupled to the back-end parts that have review comments though, so maybe I wouldn't have been able to merge anyway).